plugins/generic/pln/PLNGatewayPlugin.inc.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * @file plugins/generic/pln/PLNGatewayPlugin.inc.php
+ *
+ * Copyright (c) 2013-2014 Simon Fraser University Library
+ * Copyright (c) 2003-2014 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class PLNGatewayPlugin
+ * @ingroup plugins_generic_pln
+ *
+ * @brief Gateway component of web feed plugin
+ *
+ */
+
+import('classes.plugins.GatewayPlugin');
+import('lib.pkp.classes.site.VersionCheck');
+import('lib.pkp.classes.db.DBResultRange');
+
+class PLNGatewayPlugin extends GatewayPlugin {
+	/** @var $parentPluginName string Name of parent plugin */
+	var $parentPluginName;
+
+	function PLNGatewayPlugin($parentPluginName) {
+		parent::GatewayPlugin();
+		$this->parentPluginName = $parentPluginName;
+	}
+
+	/**
+	 * Hide this plugin from the management interface (it's subsidiary)
+	 */
+	function getHideManagement() {
+		return true;
+	}
+
+	/**
+	 * Get the name of this plugin. The name must be unique within
+	 * its category.
+	 * @return String name of plugin
+	 */
+	function getName() {
+		return 'PLNGatewayPlugin';
+	}
+
+	function getDisplayName() {
```

Comment:
Don't forget the function headers -- though those were probably missing from the code you based this from as well. The best way to do it (this is much more consistent in master) is...
    /**
     * @copydoc Plugin::getName
     */